Files/MultilingualResources/Files.tr-TR.xlf

```diff
@@ -348,27 +348,27 @@
         </trans-unit>
         <trans-unit id="WelcomeDialog.Title" translate="yes" xml:space="preserve">
           <source>Welcome to Files!</source>
-          <target state="new">Welcome to Files!</target>
+          <target state="new">Files'a hoş geldiniz!</target>
```

Comment:
Can you change the state of translated strings to "Translated"? For more information on the different states, check out the [documentation](https://files-community.github.io/docs/#/articles/translating-files?id=different-states).